spec/models/translation_override_spec.rb

```diff
@@ -96,4 +96,33 @@ def cached_value(guardian, types_name, name_key, attribute)
       include_examples "resets site text"
     end
   end
+
+  describe '#transform_pluralized_key' do
+    let(:translation_override) { TranslationOverride.new }
+    it "transforms key that end with a pluralization to end with 'one'" do
+      transformed_key = 'js.new_topics.one'
+      key = translation_override.send(:transform_pluralized_key, "js.new_topics.few")
```

Comment:
I'm not a fan of testing private methods. The `transform_pluralized_key` method should just be an implementation detail. It would be better to test that `upsert!` works, otherwise it would be quite easy to break the functionality without us noticing.

Reply:
Yeah fair. I'll do that.